orderer/common/follower/follower_chain.go

```diff
@@ -20,7 +24,12 @@ import (
 
 //TODO skeleton
 
-//go:generate mockery -dir . -name LedgerResources -case underscore -output mocks
+const (
+	defaultPullRetryMinInterval time.Duration = 10 * time.Millisecond
```

Comment:
An initial try of 10ms seems a little aggressive to me?  With an exponential backoff base of 1.2, that means we're going to have 17 retries within the first second.

Reply:
Well, if the concern is the 17 steps from 10ms to 1s, then the min interval is not the only problem, but the base is... so I'll increase both to 50ms and 1.5. This will get us from 50 ms to 60s in ~17 steps.